sys-cluster/teleport/teleport-4.0.6.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit golang-build systemd
+
+DESCRIPTION="Modern SSH server for teams managing distributed infrastructure"
+HOMEPAGE="https://gravitational.com/teleport"
+
+EGO_PN="github.com/gravitational/${PN}/..."
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3 golang-vcs
+	EGIT_REPO_URI="https://github.com/gravitational/${PN}.git"
+else
+	inherit golang-vcs-snapshot
+	SRC_URI="https://github.com/gravitational/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm"
+fi
+
+IUSE="pam"
+LICENSE="Apache-2.0"
```

Comment:
There's been a lot of talk about LICENCEs in Go packages lately. Apparently you need to include licenses for every bundled app/library, could you check and update this? 
https://packages.gentoo.org/packages/dev-util/ninka can help with it.

Reply:
Fixed.  glice (https://github.com/ribice/glice) is a good tool to get summaries of golang licenses:

```
~/go/src/github.com/gravitational/teleport git:(master) $ glice -p "github.com/gravitational/teleport"
+----------------------------------------------------------+---------------------------------------------+--------------+
|                        DEPENDENCY                        |                   REPOURL                   |   LICENSE    |
+----------------------------------------------------------+---------------------------------------------+--------------+
| github.com/gravitational/trace                           | https://github.com/gravitational/trace      | Apache-2.0   |
| github.com/gravitational/roundtrip                       | https://github.com/gravitational/roundtrip  | Apache-2.0   |
| github.com/jonboulle/clockwork                           | https://github.com/jonboulle/clockwork      | Apache-2.0   |
| github.com/sirupsen/logrus                               | https://github.com/sirupsen/logrus          | MIT          |
| golang.org/x/crypto/ssh                                  | https://go.googlesource.com/crypto          |              |
| golang.org/x/crypto/ssh/agent                            | https://go.googlesource.com/crypto          |              |
| github.com/beevik/etree                                  | https://github.com/beevik/etree             | Other        |
| github.com/coreos/go-oidc                                | https://github.com/coreos/go-oidc           | Apache-2.0   |
| github.com/golang/protobuf                               | https://github.com/golang/protobuf          | bsd-3-clause |
| github.com/gravitational/form                            | https://github.com/gravitational/form       | Apache-2.0   |
| github.com/julienschmidt/httprouter                      | https://github.com/julienschmidt/httprouter | bsd-3-clause |
| github.com/pborman/uuid                                  | https://github.com/pborman/uuid             | bsd-3-clause |
| github.com/pquerna/otp                                   | https://github.com/pquerna/otp              | Apache-2.0   |
| github.com/prometheus/client_golang                      | https://github.com/prometheus/client_golang | Apache-2.0   |
| github.com/russellhaering/gosaml2                        | https://github.com/russellhaering/gosaml2   | Apache-2.0   |
| github.com/tstranex/u2f                                  | https://github.com/tstranex/u2f             | MIT          |
| github.com/vulcand/predicate                             | https://github.com/vulcand/predicate        | Apache-2.0   |
| golang.org/x/crypto/bcrypt                               | https://go.googlesource.com/crypto          |              |
| golang.org/x/net/http2                                   | https://go.googlesource.com/net             |              |
| google.golang.org/grpc                                   |                                             |              |
| google.golang.org/grpc/codes                             |                                             |              |
| google.golang.org/grpc/credentials                       |                                             |              |
| google.golang.org/grpc/keepalive                         |                                             |              |
| google.golang.org/grpc/peer                              |                                             |              |
| github.com/gogo/protobuf                                 | https://github.com/gogo/protobuf            | Other        |
| golang.org/x/net/context                                 | https://go.googlesource.com/net             |              |
| gopkg.in/check.v1                                        |                                             |              |
| github.com/armon/go-radix                                | https://github.com/armon/go-radix           | MIT          |
| github.com/aws/aws-sdk-go                                | https://github.com/aws/aws-sdk-go           | Apache-2.0   |
| go.etcd.io/etcd/clientv3                                 | https://github.com/etcd-io/etcd             | Apache-2.0   |
| go.etcd.io/etcd/etcdserver/api/v3rpc/rpctypes            | https://github.com/etcd-io/etcd             | Apache-2.0   |
| go.etcd.io/etcd/mvcc/mvccpb                              | https://github.com/etcd-io/etcd             | Apache-2.0   |
| google.golang.org/grpc/status                            |                                             |              |
| cloud.google.com/go/firestore                            | https://code.googlesource.com/gocloud       |              |
| cloud.google.com/go/firestore/apiv1/admin                | https://code.googlesource.com/gocloud       |              |
| google.golang.org/api/option                             |                                             |              |
| google.golang.org/genproto/googleapis/firestore/admin/v1 |                                             |              |
| github.com/mattn/go-sqlite3                              | https://github.com/mattn/go-sqlite3         | MIT          |
| github.com/google/btree                                  | https://github.com/google/btree             | Apache-2.0   |
| github.com/codahale/hdrhistogram                         | https://github.com/codahale/hdrhistogram    | MIT          |
| github.com/docker/docker                                 | https://github.com/docker/docker            | Apache-2.0   |
| github.com/gokyle/hotp                                   | https://github.com/gokyle/hotp              | isc          |
| golang.org/x/crypto/ssh/terminal                         | https://go.googlesource.com/crypto          |              |
| gopkg.in/yaml.v2                                         |                                             |              |
| k8s.io/apimachinery/pkg/util/yaml                        |                                             |              |
| cloud.google.com/go/storage                              | https://code.googlesource.com/gocloud       |              |
| google.golang.org/api/iterator                           |                                             |              |
| github.com/davecgh/go-spew                               | https://github.com/davecgh/go-spew          | isc          |
| github.com/kylelemons/godebug                            | https://github.com/kylelemons/godebug       | Apache-2.0   |
| k8s.io/api/certificates/v1beta1                          |                                             |              |
| k8s.io/apimachinery/pkg/apis/meta/v1                     |                                             |              |
| k8s.io/apimachinery/pkg/fields                           |                                             |              |
| k8s.io/apimachinery/pkg/watch                            |                                             |              |
| k8s.io/client-go/kubernetes                              |                                             |              |
| k8s.io/client-go/tools/clientcmd                         |                                             |              |
| k8s.io/client-go/tools/clientcmd/api                     |                                             |              |
| github.com/gravitational/oxy                             | https://github.com/gravitational/oxy        | Apache-2.0   |
| github.com/gravitational/ttlmap                          | https://github.com/gravitational/ttlmap     | Apache-2.0   |
| k8s.io/apimachinery/pkg/api/errors                       |                                             |              |
| k8s.io/apimachinery/pkg/runtime                          |                                             |              |
| k8s.io/apimachinery/pkg/runtime/serializer               |                                             |              |
| k8s.io/apimachinery/pkg/util/httpstream                  |                                             |              |
| k8s.io/apimachinery/pkg/util/httpstream/spdy             |                                             |              |
| k8s.io/apimachinery/pkg/util/net                         |                                             |              |
| k8s.io/apimachinery/pkg/util/remotecommand               |                                             |              |
| k8s.io/apimachinery/third_party/forked/golang/netutil    |                                             |              |
| k8s.io/client-go/rest                                    |                                             |              |
| k8s.io/client-go/tools/remotecommand                     |                                             |              |
| k8s.io/client-go/transport/spdy                          |                                             |              |
| k8s.io/client-go/util/exec                               |                                             |              |
| github.com/mailgun/timetools                             | https://github.com/mailgun/timetools        | Apache-2.0   |
| github.com/mailgun/ttlmap                                | https://github.com/mailgun/ttlmap           | Apache-2.0   |
| github.com/ghodss/yaml                                   | https://github.com/ghodss/yaml              | Other        |
| github.com/gravitational/configure                       | https://github.com/gravitational/configure  | Apache-2.0   |
| github.com/russellhaering/goxmldsig                      | https://github.com/russellhaering/goxmldsig | Apache-2.0   |
| github.com/kardianos/osext                               | https://github.com/kardianos/osext          | bsd-3-clause |
| github.com/kr/pty                                        | https://github.com/kr/pty                   | MIT          |
| github.com/coreos/go-semver                              | https://github.com/coreos/go-semver         | Apache-2.0   |
| github.com/gravitational/kingpin                         | https://github.com/gravitational/kingpin    | MIT          |
| github.com/json-iterator/go                              | https://github.com/json-iterator/go         | MIT          |
| golang.org/x/net/websocket                               | https://go.googlesource.com/net             |              |
| golang.org/x/text/encoding                               | https://go.googlesource.com/text            |              |
| golang.org/x/text/encoding/unicode                       | https://go.googlesource.com/text            |              |
| github.com/mailgun/lemma                                 | https://github.com/mailgun/lemma            | Apache-2.0   |
| github.com/dustin/go-humanize                            | https://github.com/dustin/go-humanize       | Other        |
| github.com/gizak/termui                                  | https://github.com/gizak/termui             | MIT          |
| github.com/prometheus/client_model                       | https://github.com/prometheus/client_model  | Apache-2.0   |
| github.com/prometheus/common                             | https://github.com/prometheus/common        | Apache-2.0   |
| github.com/google/gops                                   | https://github.com/google/gops              | bsd-3-clause |
+----------------------------------------------------------+---------------------------------------------+--------------+

```